README.md

```diff
@@ -30,7 +30,7 @@ in our IR, we don’t have any variables, their versions and name. Everything
 is represented by computation Nodes and Edges between them. Of course, we
 have special PHI Node that represents the Phi() function.
 
-Despite, our IR is graph based, internally, it’s represented as a plain
+Internally, our IR is graph-based and represented as a plain
```

Comment:
This changes the meaning of the original sentence. Graph based IR usually assume independently allocated nodes with direct pointers between them. Usage of plain arrays for that is an uncommon decision.
May be this should be **Internally, our graph-based IR is represented as a plain array**

Reply:
I'm sorry! I've changed this as you proposed.